runtime-modules/content/src/benchmarks/mod.rs

```diff
@@ -937,6 +939,134 @@ where
     Ok(new_group_id)
 }
 
+pub fn all_channel_pausable_features_except(
+    except: BTreeSet<crate::PausableChannelFeature>,
+) -> BTreeSet<crate::PausableChannelFeature> {
+    [
+        crate::PausableChannelFeature::ChannelFundsTransfer,
+        crate::PausableChannelFeature::CreatorCashout,
+        crate::PausableChannelFeature::ChannelUpdate,
+        crate::PausableChannelFeature::VideoNftIssuance,
+        crate::PausableChannelFeature::VideoCreation,
+        crate::PausableChannelFeature::ChannelUpdate,
+        crate::PausableChannelFeature::CreatorTokenIssuance,
+    ]
+    .iter()
+    .filter(|&&x| !except.contains(&x))
+    .map(|&x| x)
+    .collect::<BTreeSet<_>>()
+}
+
+pub fn create_pull_payments_with_reward<T: Config>(
+    payments_number: u32,
+    cumulative_reward_earned: BalanceOf<T>,
+) -> Vec<PullPayment<T>> {
+    let mut payments = Vec::new();
+    for i in 1..payments_number {
```

Comment:
`1..payments_number` range will only cover `payments_number - 1` numbers, for example, if `payments_number` is `2` the range will only contain `[1]`, if the `payments_number` is `4` is will be `[1, 2, 3]` etc.
You should use `1..=payments_number` instead to make the range inclusive